website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+
+#### Create New File button
+This button allows the user to create a new file in their project. 
+
+## Editing 
+
+The layout of the IDE editing page is divided into the following areas, and more features.
+
+A &mdash; [File Editor](#file-editor)<br />
+B &mdash; [Interaction bar](#interaction-bar)<br />
+C &mdash; [Preview button](#preview-button)<br />
+D &mdash; [Compile button](#compile-button)<br />
+E &mdash; [Build button](#build-button)<br />
+F &mdash; [Results subtab](#results-subtab)<br />
+G &mdash; [Compiled code subtab](#compiled-code-subtab)<br />
+H &mdash; [Lineage subtab](#lineage-subtab)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-editing.jpg" width="100%" title="The Cloud IDE editing components"/>
+
+#### File Editor
+The File Editor is the region where users edit code. This region is broken out by tabs for all files that have been opened. Unsaved Files are marked with a blue dot icon in the tab view. 
+
+The editor has a **Format** button enabled in this area for `{{  filetypes  }}` files, and will use your selected formatter (sqlfmt or sqlfluff) to edit the file contents. 
+
+#### Interaction bar
+This interaction bar section sits below the File editor and contains all the sub-tabs and buttons that help you preview, compile, build, view the DAG, and more. 
+
+Refer to bullets C to H above for more details on the sub-tabs and button details. 
+
+#### Preview button 
+The Preview Button executes the SQL in the active file editor (regardless of save status) against your warehouse, and returns the results to the Results subtab. Queries executed via the Preview Button will automatically have a limit 500 appended to the end to prevent returning excessive data to the IDE and causing browser issues. 
+
+#### Compile button
+The Compile Button takes the contents of the active File Editor (regardless of save status) and returns the compiled SQL to the Compiled Code Subtab
+
+#### Build button 
+The build button is a shortcut for the user to invoke dbt commands anchored by the active model in the File Editor. The available commands are dbt build, dbt test, and dbt run with options to include only the current resource, the resource and its upstream dependencies, the resource and its downstream dependencies, or the resource with all dependencies. This menu is enabled for all executable nodes.
+
+#### Results subtab
+The Results subtab displays the most recent Preview results in tabular format. 
+
+#### Compiled code subtab
+The Compiled Code Tab shows the compiled code generated by the most recent compile invocation triggered by the Compile button.
+
+#### Lineage subtab
+The Lineage subtab shows the lineage for the active model in the File Editor. By default, this menu will display 2 degrees of lineage in both directions (`2+model_name+2`). This can be edited to display any desired selection syntax. 
+
+#### Minimap
```

Comment:
I find it hard to link these sections to the screenshot above, trying to match letters (A, B, C) which don't exist in the sections here to some which do exist above and inside the screenshot.

I wonder if small screenshots in these individual sections which are just cropped to the relevant content may help disambiguate?

Reply:
i dig it!